staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -88,6 +88,14 @@ const (
 	// audited.
 	AdvancedAuditing featuregate.Feature = "AdvancedAuditing"
 
+	// owner: @logicalhan
+	// kep: https://kep.k8s.io/3466
+	// alpha: v1.26
+	// beta: v1.27
+	//
+	// Enables component SLIs metrics endpoint (i.e. `/metrics/slis`)
+	ComponentHealthSLIs featuregate.Feature = "ComponentHealthSLIs"
```

Comment:
I thought you were putting these in more than one binary, isn't there a more general place for such features?

Reply:
Alpha actually intentionally only targets apiserver as per the KEP.

Reply:
Yeah, that makes sense. Done.